hail_search/queries/mito.py

```diff
@@ -112,6 +118,215 @@ class MitoHailTableQuery(BaseHailTableQuery):
     }
     SORTS[PATHOGENICTY_HGMD_SORT_KEY] = SORTS[PATHOGENICTY_SORT_KEY]
 
+    def _import_and_filter_entries_ht(
+        self, project_guid: str, num_families: int, project_sample_type_data, **kwargs
+    ) -> tuple[hl.Table, hl.Table]:
+        sample_types = set(project_sample_type_data.keys())
+        if len(sample_types) == 1:
+            return super()._import_and_filter_entries_ht(project_guid, num_families, project_sample_type_data, **kwargs)
+
+        self._has_both_sample_types = True
+        entries_hts_map: dict[str, list[tuple[hl.Table, dict]]] = {}
+        for sample_type in sample_types:
+            if num_families == 1:
+                family_guid = list(list(project_sample_type_data.values())[0].keys())[0]
+                ht, sample_data = self._load_prefiltered_family_ht(family_guid, sample_type, project_sample_type_data, **kwargs)
+            else:
+                ht, sample_data = self._load_prefiltered_project_ht(project_guid, sample_type, project_sample_type_data, **kwargs)
+            entries_hts_map[sample_type] = [(ht, sample_data)]
+
+        return self._filter_entries_ht_both_sample_types(entries_hts_map, **kwargs)
+
+    def _filter_entries_ht_both_sample_types(
+        self, entries_hts_map: dict[str, list[tuple[hl.Table, dict]]], n_partitions=MAX_PARTITIONS,
+        inheritance_filter=None, quality_filter=None, **kwargs
+    ):
+        filtered_hts = []
+        filtered_comp_het_hts = []
+        wes_entries, wgs_entries = entries_hts_map[SAMPLE_TYPE_WES], entries_hts_map[SAMPLE_TYPE_WGS]  # entries_hts_map: {<sample_type>: [(ht, project_samples), ...]}
+
+        # Iterate over each chunk of project tables
+        for (wes_ht, wes_project_samples), (wgs_ht, wgs_project_samples) in zip(wes_entries, wgs_entries):
```

Comment:
as noted above, it makes more sense to not have this as a list as you are only ever considering one table for each sample type in this function

Reply:
this function is used in the flow (called above) where there is only one table per sample type as well as for when there may be multiple tables per sample type